v1/authorization/scripts/create_admin.ts

```diff
@@ -0,0 +1,15 @@
+import { readFileSync } from 'fs'
+import path from 'path'
+import {createSigner} from 'fast-jwt'
+import { fileURLToPath } from 'url';
+import { makeAdmin } from '../jwt.js'
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const secret = readFileSync(path.join(__dirname, '..', 'keys', 'private.key'), 'utf8')
```

Comment:
We should have a way to configure this path. Either an env variable or a CLI flag.